src/sensors/package.xml

```diff
@@ -2,10 +2,10 @@
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
   <name>sensors</name>
-  <version>0.0.0</version>
-  <description>TODO: Package description</description>
+  <version>0.0.1</version>
+  <description>ROS 2 IMU subscriber package for logging IMU data to a CSV file.</description>
   <maintainer email="shandilya.shreyas@gmail.com">root</maintainer>
-  <license>TODO: License declaration</license>
+  <license>Apache-2.0</license>
```

Comment:
Can you also take up the task of updating the license in all packages that we building? this will ensure safety of our software.

Reply:
sure.